weblate/formats/models.py

```diff
@@ -24,6 +24,28 @@
 from weblate.utils.classloader import ClassLoader
 
 
+class ExporterLoader(ClassLoader):
+    def __init__(self):
+        super().__init__("WEBLATE_EXPORTERS", False)
+        self.errors = {}
+
+
+EXPORTERS = ExporterLoader()
+
+
+def get_exporter(name):
+    """Return registered exporter."""
+    return EXPORTERS[name]
```

Comment:
Maybe this is no longer needed? I don't see reason why not to use `EXPORTERS[name]` directly.

Reply:
Makes sense.